### PR TITLE
Added codechef to the challenges section

### PR DESCRIPTION
**Pull request by Akshita-Sharma-25**
## Summary:

Added:

- one
- two
- three

### CODE REVIEW

**Comment by Akshita-Sharma-25** [comment]
added codechef to challenges section

File: README.md [hunk 1]
```diff
@@ -111,6 +111,7 @@ vous trouverez sur cette plateform des cours et examens qui vous accompagnerons
 * [Exercism](http://exercism.io/) Code practice and mentorship for everyone
 * [w3schools](https://www.w3schools.com/) W3Schools is optimized for learning, testing, and training.
 
+
```
Comment (Fcmam5) [inline README.md:114]:
```suggestion
```

**Review by Fcmam5 — APPROVED** [review]

**Comment by Fcmam5** [comment]
Thank you @Akshita-Sharma-25 for your contribution!

**Comment by Akshita-Sharma-25** [comment]
Hope my contribution benefits others

**Pull request merged**